server/website/script/fixture_generators/knob_settings/oracle/create_knob_settings.py

```diff
@@ -0,0 +1,210 @@
+#
+# OtterTune - create_knob_settings.py
+#
+# Copyright (c) 2017-18, Carnegie Mellon University Database Group
+#
+import json
+import shutil
+
+# Oracle Type:
+# 1 - Boolean
+# 2 - String
+# 3 - Integer
+# 4 - Parameter file
+# 5 - Reserved
+# 6 - Big integer
+
+
+# Ottertune Type:
+# STRING = 1
+# INTEGER = 2
+# REAL = 3
+# BOOL = 4
+# ENUM = 5
+# TIMESTAMP = 6
+
+# miss:
+# OPTIMIZER_MODE
+# cursor_sharing
+
+
+def set_field(fields):
+    if fields['name'].upper() == 'MEMORY_TARGET':
```

Comment:
why are you doing this boring if conditions when you could define a dict as:
```
oracle_settings = {'MEMORY_TARGET: {'tunable': False, 'minval': 0, 'maxval':'33000000000', 'default' = 0},
                               'MEMORY_MAX_TARGET: {'tunable': False, 'minval': 0, 'maxval':'33000000000', 'default' = 0},
                               ...}
```
Then on set_field(fields) it would be:
```
def set_field(fields):
     oracle_fields_u = oracle_settings.get(fields['name'].upper())
     oracle_fields_l = oracle_settings.get(fields['name'].lower())   # cause some keys are lowercase
     if oracle_fields_u:
        fields.update(oracle_fields_u)
     elif oracle_fields_l:
        fields.update(oracle_fields_l)
```

One advantage of this approach is that the variables could be stored on a separate json or yaml file, so easily changed without needing to change the Python code, for example.